content/articles/compiled-redesign.md

```diff
@@ -0,0 +1,141 @@
+
+### Semantic HTML and ARIA
+
+In her demonstration on 
+[how a screen reader user accesses the web](https://www.smashingmagazine.com/2019/02/accessibility-webinar/),
+accessibility engineer [Léonie Watson](https://tink.uk/) recommended using
+semantic HTML and ARIA attributes to improve content accessibility and clarify
+meaning for content groups. They serve as sign posts to help users to chart a
+mental model of a page. I was already using HTML5 landmark regions (`header`,
+`nav`, `main`, `section`, `article`, `footer`) in _CompilED_. This time, I paid
+more attention to the ARIA labels to identify components on the pages so that
+they can be read by screen readers. 
+
+### Keyboard accessibility
+
+Keyboard navigation is markedly improved for _CompilED_ pages—all functionality
+and interactive elements are available from a keyboard. I also redesigned the
+visual focus styling (`hover`, `focus`, `active`), thanks to the
+[helpful tips from Eric Bailey.](https://www.youtube.com/watch?v=9YazmVNwtHI)
+
+### Alt text for images and social media
+
+While it is common knowledge that alternative text for images is necessary,
+what is less know is what sort of description should be used for the `alt`
+text. To put simply, image description depends on context.
+
+All images on _CompilED_ articles how have `alt` attribute built in the
+template. It is up to the authors to decide how the images are used to provide
+context in the `alt` text. W3C provides a good
+[tutorial on image concepts](https://www.w3.org/WAI/tutorials/images/).
+WebAIM also has a
+[thorough explanation on alternative text](https://webaim.org/techniques/alttext/)
+and images.
+
+_CompilED_ also uses `og:image:alt` and `twitter:image:alt` structured
+properties to provide descriptions images used for social media sharing.
+
+### Audits and limitations
+
+I used the following scanning tools to test accessibility compliance for
+_CompilED_:
+
+*
+[Siteimpove](https://chrome.google.com/webstore/detail/siteimprove-
+accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)
+accessibility checker, by [Siteimprove](https://siteimprove.com)
+* [Wave](https://wave.webaim.org/extension/) web accessibility evaluation tool
+by [WebAIM](https://webaim.org/)
+* [Axe](https://www.deque.com/axe/) web accessibility testing tool by 
+[Deque Systems](https://www.deque.com)
+* [VoiceOver](https://www.apple.com/accessibility/mac/vision/), the screen
+reader by Apple
+
+The scan shows some color contrast errors, but for the most part, _CompilED_
+[conforms to WCAG 2.0 Level AA](/info/accessibility/). 
+
+## Coda
+
+Web content accessibility isn’t just about technology. _CompilED_ could have
+passed all the scanning audits and still wouldn’t be very usefully if I didn’t
+understand the user experience with screen readers or other assistive
+technologies. We should make POUR (Perceivable, Operable, Understandable,
+Robust) design regarding digital experience and accessibility for _people of
+all abilities_.
```

Comment:
Sentence starting: We should make POUR...I think a verb is needed here.

Reply:
maybe 'choices'?

Reply:
I was trying to be punny. I'm rewriting the sentence, and still be punny.